spec/rspec/mocks/verifying_doubles/construction_spec.rb

```diff
@@ -148,13 +148,13 @@ module CustomModule
 
         it 'prevents creation of instance doubles for unloaded constants' do
           expect {
-            instance_double('LoadedClas')
+            instance_double('LoadedClass')
```

Comment:
Something makes me think this is an intentional typo

Reply:
Dropped this change.